lib/auth.ts

```diff
@@ -20,6 +20,7 @@ export type AuthContext<TContext extends object = object> = {
 
 export class AuthDirectiveVisitor<
   TContext extends AuthContext,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
 > extends EasyDirectiveVisitor<{}> {
```

Comment:
If there are no args, use  `Record<string, never>`
```suggestion
> extends EasyDirectiveVisitor<Record<string, never>, TContext> {
```